### ISSUE TITLE
Question regarding DataLoader in vqvae.py

### ISSUE THREAD

**Issue by xiaodaxia007** [ritheshkumar95/pytorch-vqvae#17]
I am just wondering why your let train_loader shuffle = False and test_loader  shuffle=True? Should it be vice versa?
